tests/ferret/core_test.go

```diff
@@ -150,10 +150,16 @@ func TestCore(t *testing.T) {
 			require.NoError(t, err)
 		}
 
+		// o is FindOptions, it's used in projections testing.
+		// v is a value and it is also being used in the projections testing,
+		// since the resulting value for projection differs from the document value.
+		// It's being checked only one of (v, IDs, err).
```

Comment:
And with inline comments we no longer need that comment

Reply:
removed